http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/leoPubsubMessageSubscriberModels.scala

```diff
@@ -108,12 +114,21 @@ object LeoPubsubMessage {
     val messageType: LeoPubsubMessageType = LeoPubsubMessageType.StartRuntime
   }
 
-  final case class RuntimeFollowupDetails(runtimeId: Long, runtimeStatus: RuntimeStatus)
-      extends Product
-      with Serializable
+  final case class UpdateRuntimeMessage(runtimeId: Long,
+                                        newMachineType: Option[MachineTypeName],
+                                        stopUpdateMachineType: Boolean,
```

Comment:
what does `stopUpdateMachineType` mean? maybe add a comment?

Reply:
I can add a comment.. if `stopUpdateMachineType == true` then the runtime should go through a stop-update transition (via `FollowupDetails`). If `false` then the machine type can be changed immediately.

(This also implies that there is no more use for the `StopUpdate` message type, as `UpdateRuntimeMessage` will handle the stop-update flow.)

Reply:
shall we delete `StopUpdate` message then?

Reply:
It's still used by `LeonardoService`.. I really want to delete `LeoRoutes` and `LeonardoService` ;)

Reply:
I'm thinking once we migrate terra-ui to use new route, we can at least delete the patch implementation in leonardoservice since patch is added recently and mostly likely no one is using it

Reply:
I wonder if `stopBeforeUpdatingMachineType` or `stopToUpdateMachineType` may be more explanatory without before overly verbose  🤔

Reply:
I like `stopToUpdateMachineType`, I can change it